### ISSUE TITLE
Django 1.10.2 working?

### ISSUE THREAD

**Issue by 7wonders** [burke-software/django-mass-edit#65]
I have installed master and updated urls but I am not getting the mass edit form only a single form for the first object. Did a quick bit of debugging and everything seems to be reaching mass_change_view correctly. Any ideas?

**Comment by a741424975game** [comment]
from django.conf.urls import include, url
from django.contrib import admin
from massadmin import urls as massadmin_urls
**from massadmin import mass_change_selected

admin.site.add_action(mass_change_selected)**

urlpatterns = [
    url(r'', include(admin.site.urls)),
    url(r'', include(massadmin_urls)),
]

**Comment by 7wonders** [comment]
yep, thats exactly how its set up and with massadmin in installed apps but still only a single form.

**Comment by 7wonders** [comment]
Been a while to return to this but I needed a mass-edit solution again so. I found that when using django-guardian it will always return 'admin/guardian/model/change_form.html' with self.change_form_template line 187 in massadmin.py. Is the self.change_form_template needed? Seems to defy the point of the mass-edit override.
